python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -588,6 +588,7 @@ def emit_with_mutating_variants(key, **kwargs):
     emit("aten::any.bool : (bool[]) -> (bool)")
     emit("aten::sort.int : (int[], bool) -> ()", has_canonicalizer=True)
     emit("aten::sort : (Tensor, int, bool) -> (Tensor, Tensor)")
+    emit("aten::unbind.int : (Tensor, int) -> (Tensor[])", traits=["Pure"])
```

Comment:
This is not a Pure op, since it can raise an exception when the inputs are invalid

Reply:
I wanted to use `Pure` trait to DCE the op if there's no use. Sounds like I need a pattern for it then.